library/core/src/iter/adapters/intersperse.rs

```diff
@@ -173,8 +173,7 @@ where
 {
```

Comment:
You can remove `Peekable` from this function signature.

Reply:
I changed from `Peekable<I: Iterator>` to just `I: Iterator` only.